frontends/web/src/containers/SubmitInterface.js

```diff
@@ -110,7 +110,7 @@ class SubmitInterface extends React.Component {
       .then(
         (result) => {
           values.modelName = "";
-          for (const [fname, _] of Object.entries(files)) {
```

Comment:
Does this work? I thought Object.entries() returns a list of `key, value` pairs so `fname` might become a tuple here instead of the name of the file

Reply:
I might be wrong though..

Reply:
Yup, the code will still work because of Javascript's destructuring assignment (ES6 syntax). This image might explain what it does:
<img width="360" alt="image" src="https://user-images.githubusercontent.com/48560219/142391608-3d70ca24-36d6-4622-994a-0195cde6df4c.png">

Reply:
maybe more appropriate here would just be to use `Object.keys(files)`? I think not having an unused var at all > underscores for unused vars > skipping the variable in a destructuring assignment

Reply:
that does make sense. I'll make those changes too